### PR TITLE
Stop hardcoding file_packager.py directory

### PR DESCRIPTION
**Pull request by dalcde**
This makes our build system more flexible, especially if we start using unpatched emscripten in the future.

### CODE REVIEW

File: pyodide_build/common.py [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 
 ROOTDIR = Path(__file__).parents[1].resolve()
 TOOLSDIR = ROOTDIR / "tools"
-PACKAGERDIR = ROOTDIR / "emsdk" / "emsdk" / "fastcomp" / "emscripten" / "tools"
+
+# Use emcc.py because emcc may be a ccache symlink
+_EMCC_PATH = shutil.which("emcc.py")
+if _EMCC_PATH is None:
+    print("emcc.py not found. Setting file_packager.py path to /dev/null")
+    PACKAGERPATH = Path("/dev/null")
```
Comment (rth) [inline pyodide_build/common.py:12]: Shouldn't we rather error here?

Comment (dalcde) [reply pyodide_build/common.py:12]: That would break all pyodide_build tests

Comment (dalcde) [reply pyodide_build/common.py:12]: I could make it a function instead

Comment (rth) [reply pyodide_build/common.py:12]: Otherwise maybe keep to to None here, and then in the other place where we need it, since that's not used in test, check it if's None and error. 

I'm just not very keen on the `Path(/dev/null)` because if it's actually used somewhere it will be confusing to debug.
